packages/runtime-core/src/scheduler.ts

```diff
@@ -50,7 +51,7 @@ type CountMap = Map<SchedulerJob | SchedulerCb, number>
 
 export function nextTick(fn?: () => void): Promise<void> {
   const p = currentFlushPromise || resolvedPromise
-  return fn ? p.then(fn) : p
+  return fn ? p.then(fn.bind(currentInstance && currentInstance.proxy)) : p
```

Comment:
currentInstance?.proxy ? fn.bind(currentInstance.proxy): fn

would be better.

Reply:
I think we don't have the optional chaining set up for this project yet though.